source/documentation/api_key/api_key.md

```diff
@@ -0,0 +1,9 @@
+## Authentication 
+
+If you have created an API key then you can use it with all registers.
+
+Your API key should be of the form `76415efc-3d0f-4522-affd-bb6464e5ea88`. You should place it in requests as part of the `Authorization` header. For example, this is a valid request:
```

Comment:
Just checking this isn't a real key!

Reply:
No, it's from a GUID generator

Reply:
I would not include a realistic API key here as people might just use that instead of registering. I would rather say something like:
> Use the API key you received when registering

and then in the curl example do:

`curl https://country.register.gov.uk/record/GB.json -H "Authorization: YOUR-API-KEY-HERE"`

Reply:
@gidsg good shout but I've closed this PR; the new one is https://github.com/alphagov/registers-tech-docs/pull/43

Reply:
I would rather explicitly say:
> It should be included in all requests using the `Authorization` header